Source/WebCore/rendering/AttachmentLayout.mm

```diff
@@ -278,6 +279,24 @@ static CGFloat attachmentDynamicTypeScaleFactor()
 
 AttachmentLayout::AttachmentLayout(const RenderAttachment& attachment, AttachmentLayoutStyle)
 {
+    if (attachment.attachmentElement().isImageOnly()) {
+        iconRect = FloatRect(0, 0, attachmentIconOnlySize.width, attachmentIconOnlySize.height);
+        iconBackgroundRect = iconRect;
+        attachmentRect = encloseRectToDevicePixels(iconBackgroundRect, attachment.document().deviceScaleFactor());
+
+        icon = attachment.attachmentElement().icon();
+        if (!icon)
+            attachment.attachmentElement().requestIconWithSize(iconRect.size());
+
+        thumbnailIcon = attachment.attachmentElement().thumbnail();
+
+        hasProgress = getAttachmentProgress(attachment, progress);
+        if (hasProgress)
+            progressRect = FloatRect((attachmentRect.width() / 2) - (attachmentProgressSize / 2), (attachmentRect.height() / 2) - (attachmentProgressSize / 2), attachmentProgressSize, attachmentProgressSize);
```

Comment:
Nit: This could use `(attachmentRect.width() - attachmentProgressSize) / 2` and `(attachmentRect.height() - attachmentProgressSize) / 2`.

Reply:
I copied it from the existing code below! Maybe it was important to do the truncating divisions first? (Though I doubt it matters much, it may move by 1 pixel here&there.)
I'll change it only here in this new code.

Reply:
Oh! Maybe just keep it the same then...